Godeps/Godeps.json

```diff
@@ -0,0 +1,164 @@
+{
+	"ImportPath": "github.com/kinvolk/tcd",
+	"GoVersion": "go1.6",
+	"GodepVersion": "v66",
+	"Deps": [
+		{
+			"ImportPath": "github.com/Sirupsen/logrus",
+			"Comment": "v0.10.0-16-gcd7d1bb",
```

Comment:
For tcd, it does not matter for now, but in general, we try to use a fixed version of dependencies instead of whatever was the last commit on the master branch. It makes packaging (e.g. in Debian) easier.

Just for your information. No need to check that for now.